client/my-sites/email/inbox/mailbox-selection-list/index.jsx

```diff
@@ -138,27 +141,62 @@ const MailboxListStatus = ( { isError, statusMessage } ) => {
 
 MailboxListStatus.propType = {
 	isError: PropTypes.bool,
-	statusMessage: PropTypes.string.isRequired,
+	statusMessage: PropTypes.node.isRequired,
+};
+
+const MailboxLoaderError = ( { refetchMailboxes, siteId } ) => {
+	const translate = useTranslate();
+	const queryClient = useQueryClient();
+
+	const reloadMailboxes = () => {
+		queryClient.removeQueries( getMailboxesCacheKey( siteId ) );
+		refetchMailboxes();
+	};
+
+	return (
+		<div className="mailbox-selection-list__loader-error-container">
+			<section>
+				<img src={ unsupportedBrowserIllustration } alt={ translate( 'Loading error icon' ) } />
+				<FormattedHeader
+					align="center"
+					brandFont
+					headerText={ translate( "Something's broken" ) }
+					subHeaderText={ translate(
+						'There has been an error in loading your mailboxes. Please use the reload button, and reach out to support if the issue persists.'
+					) }
+				/>
+				<footer>
+					<Button primary onClick={ reloadMailboxes }>
+						{ translate( 'Reload mailboxes' ) }
+					</Button>
+					<Button borderless primary href={ CALYPSO_CONTACT }>
```

Comment:
Without the border, this button doesn't look much like a link or clickable item until I mouse over. I'd prefer if we had a border here, but I assume @saygunnyc's design removed this. (This is _not_ a blocker for shipping at all! This is 100% personal preference!)